docs/concepts.rst

```diff
@@ -415,13 +415,44 @@ explictly specify a serialization strategy for an entity by attaching a
 
 .. _Protocol: api/protocols.rst
 
-Exporting Persisted Files
-.........................
+Retrieving Persisted Files
+..........................
+In some cases, you'll want to directly access the persisted file(s) for an
+entity rather than its in-memory representation.  (For example, if you're
+writing a paper or report, you may want to access the files containing the
+plots.) This can be achieved using the :meth:`Flow.get <bionic.Flow.get>`
+method. Its ``collection`` parameter specifies whether to return a single value
+or, if more than one, the wrapping data structure.  Its ``mode`` parameter
+indicates the type of object to represent each persisted file.  For example:
+
+.. code-block:: python
+
+    flow = builder.build().setting('subject', values=['Alice', 'Bob'])
+    flow.get('subject', collection=list, mode=Path)
+
+Would return a ``List[Path]``, one ``Path`` per persisted file (in this case, 2).
```

Comment:
style nit: Treating the code block as part of the sentence suggests that we shouldn't capitalize "would".  Maybe the cleanest approach is to write "This would ..."?

Reply:
SGTM!